### PR TITLE
small changes to env variables

### PR DESCRIPTION
**Pull request by caitlinturnidge**
Really small changes to get the code working in a container and on the cloud!

### CODE REVIEW

File: s3_data_management/s3_data_management.py [hunk 1]
```diff
@@ -17,16 +17,16 @@
...
+                  aws_secret_access_key=environ['AWS_SECRET_ACCESS_KEY_'])
 
 
 def get_bucket_keys(s3_client: client, folder_path: str, bucket_name: str) -> list:
     """Returns a list of keys of csv files, with a prefix matching the folder_path."""
     objects = s3_client.list_objects(
         Bucket=bucket_name, Prefix=folder_path).get('Contents')
     if objects:
-        return [obj['Key'] for obj in objects if '.csv' in obj['Key']]
+        return [obj['Key'] for obj in objects if obj['Key'].split('.')[-1] == 'csv']
```
Comment (Shurc1val) [inline s3_data_management/s3_data_management.py:29]: why this change?

Comment (caitlinturnidge) [reply s3_data_management/s3_data_management.py:29]: I was fiddling around earlier, but I don't think this actually makes a difference now

Comment (Shurc1val) [reply s3_data_management/s3_data_management.py:29]: No worries, I'll approve

**Review by Shurc1val — APPROVED** [review]
Needs must; @haydenjack you may have to do something similar for pipeline script; will add a ticket to trello to sort this out better if we get a chance.
